utils/readsb/Makefile

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2020 Stijn Tintel <stijn@linux-ipv6.be>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=readsb
+PKG_VERSION:=3.8.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/Mictronics/readsb/archive/v$(PKG_VERSION)
+PKG_HASH:=bf59b8ecd0ff66945b210c55a5b824aa63ff8cbb1704249528d30a4902e716b2
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+PKG_BUILD_DEPENDS:=!USE_GLIBC:argp-standalone
```

Comment:
No idea if PKG_INSTALL would work. PKG_BUILD_PARALLEL probably would.

Reply:
I don't see the need for PKG_INSTALL. It's just 2 binaries in the root of the source. Using PKG_INSTALL would only result in a bigger Makefile.

Added PKG_BUILD_PARALLEL.